src/OnboardingSPA/components/LanguageSelection/index.js

```diff
@@ -0,0 +1,30 @@
+import { memo } from '@wordpress/element';
+
+const LanguageSelection = ( {
+	labgeageSelectionLabel,
```

Comment:
I think there's a misspelling here in 'labgeage'

Reply:
yup ... fixed!